pkg/utils/utils.go

```diff
@@ -138,3 +139,26 @@ func CheckSupportedArgs(supportedArgs []string, args map[string]interface{}) err
 	}
 	return nil
 }
+
+func ValidateLabels(labels map[string]string) error {
```

Comment:
I always get sad when things get dumped into generic `utils` - there already is a `validate` package, maybe it would be a good fit there?